Source/Csla.Shared/Reflection/ServiceProviderMethodCaller.cs

```diff
@@ -403,14 +403,17 @@ public static async Task<object> CallMethodTryAsync(object obj, ServiceProviderM
         int index = 0;
         int criteriaIndex = 0;
 #if !NET40 && !NET45
-        var service = ApplicationContext.ScopedServiceProvider;        
+        var service = ApplicationContext.ScopedServiceProvider;   
+        if(service == null)
```

Comment:
This qualifies as an unrelated change - let me know if you want it on another PR.

Reply:
Sorry I missed this comment about throwing an nre.. yes I hadn't realise that would avoid the need for a translation, would have saved me a headache